requirements/base.txt

```diff
@@ -1,7 +1,8 @@
 chevron~=0.12
 click~=7.0
 Flask~=1.0.2
-boto3~=1.10, >=1.10.29
+#Need to add Schemas latest SDK.
+boto3~=1.12, >=1.12.X
```

Comment:
Sorry for confusing comment, `x` was an example. Are you sure `1.12.X` works?

Reply:
I am waiting on new SDK will update.